iotedgedev/utility.py

```diff
@@ -188,3 +189,24 @@ def nested_set(self, dic, keys, value):
             current = current.get(key)
 
         current[keys[-1]] = value
+
+    @staticmethod
+    @suppress_all_exceptions()
+    def hash_connection_str_hostname(connection_str_hostname):
```

Comment:
Seems like this could be in the ConnectionString class.

Reply:
Moved. Thanks.